rocAL/rocAL/source/pipeline/image.cpp

```diff
@@ -175,6 +187,23 @@ void Image::update_image_roi(const std::vector<uint32_t> &width, const std::vect
     }
 }
 
+void Image::update_image_orig_roi(const std::vector<uint32_t> &width, const std::vector<uint32_t> &height)
+{
+    if(width.size() != height.size())
+        THROW("Batch size of image height and width info does not match")
+
+    if(width.size() != info().batch_size())
+        THROW("The batch size of actual image height and width different from image batch size "+ TOSTR(width.size())+ " != " +  TOSTR(info().batch_size()))
+    if(! _info._orig_roi_width || !_info._orig_roi_height)
```

Comment:
Please remove additional space after ! operator

Reply:
Resolved in the latest commit